src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function canAcceptValue (element) {
```

Comment:
@georgiy-abbasov hmm... if not `input` than probably `editor`?
`isEditorElement` ? @inikulin